src/test/java/com/google/api/codegen/gapic/testdata/nodejs/nodejs_library.baseline

```diff
@@ -351,36 +320,19 @@ function sampleFindRelatedBooks() {
 }
 
 sampleFindRelatedBooks();
-*/
 // [END sample_core]
-
-// FIXME: Insert here clean-up code.
 // tslint:disable-next-line:no-any
 // [END sample]
 ============== file: src/samples/v1/find_related_books_request_async_paged_odyssey.js ==============
 //// [ This is an auto-generated sample file produced by the gapic-generator. Sample name: "FindRelatedBooksRequestAsyncPagedOdyssey" ]
-//// STUB standalone sample "FindRelatedBooksRequestAsyncPagedOdyssey" /////
-
-// FIXME: Insert here set-up comments that we never want to display in cloudsite. These are seen by users perusing the samples directly in the repository.
-
+'use strict';
 // [START sample]
-
-// FIXME: Insert here boilerplate code not directly related to the method call itself.
-
-//      calling form: "RequestAsyncPaged"
-//        region tag: "sample"
-//         className: "FindRelatedBooksRequestAsyncPagedOdyssey"
-//          valueSet: "odyssey" ("The Odyssey")
-//       description: "Testing calling forms"
-//        [names[0]=Odyssey, shelves[0]=Classics]
-//      apiMethod "findRelatedBooks" of type "PagedOptionalArrayMethod"
-
 // [START sample_core]
-
-// FIXME: Insert here code to prepare the request fields, make the call, process the response.
-
-/*
 function sampleFindRelatedBooks() {
+  const library = require('@google-cloud/library');
```

Comment:
@alexander-fenster @beccasaurus 
Currently this `require('@google-cloud/library')` is not compiling, since the samples and the client live in the same package, and they are sharing a common collection dependencies (`package.json`).
To make it work, we can:
1. Change `require('@google-cloud/library')` to `require('../../src')` similar to smoke test, but that would be very confusing for people who read the samples.
2. Move the samples out to a separate package which has it's own `package.json`. In this case we can keep `require('@google-cloud/library')`, and with some post-processing, it would be fairly easy to have the samples depend on either the generated client or a previous version that has already been published. However this would take fairly longer time for us to implement, as we might need to change artman too to make this happen.

Reply:
Option 2 is how it works now. We generate a fake `package.json` for samples and put it inside `samples/` folder. No need for post-processing, all the logic you're talking about is being taken care of by `npm link` if needed.

Reply:
Cool!